### ISSUE TITLE
Recaptcha On Register Page

### ISSUE THREAD

**Issue by TylerL95** [StudioForty9/Recaptcha#2]
Hi, im working on magento 1.9 and im having an issue getting my recaptcha to work on my website. It works perfectly on the contacts page but it doesnt seem to work on the register page

**Comment by studioforty9-dev** [comment]
Hi Tyler,

Which version are you using?

**Comment by TylerL95** [comment]
Hi,
Im using the latest version of the plugin (1.1.0)

**Comment by studioforty9-dev** [comment]
Hi Tyler,

The latest version is v1.2.0 but that's okay, did you install it via composer? Have you added the code snippet:

``` php
<?php echo $this->getChildHtml('studioforty9.recaptcha.explicit'); ?>
```

on the customer registration template `app/design/frontend/template/persistent/customer/form/register.phtml`?

Have you enabled the widget via configuration
- 'System => Configuration => Sales => Google API => Google ReCaptcha => Enabled
- 'System => Configuration => Sales => Google API => Google ReCaptcha => Enabled on Customer Registration Form'

**Comment by TylerL95** [comment]
i installed it via magento connect 
http://www.magentocommerce.com/magento-connect/recaptcha-1.html

ive put the code in but the recaptcha is still not showing,There is also no "Enabled on Customer Registration Form" in the backend

**Comment by studioforty9-dev** [comment]
Okay, you don't have v1.1.0 then. Not your fault. Magento are having trouble with our extension, they're listing the only stable version as v1.0.1 which is wrong. I've contacted them 6 times this week but they haven't replied. 

If you open up the Magento Connect Manager again, you can uninstall the version you have and upload and install this tarball - http://cl.ly/1r1C3N1A3f3z

That'll get you up and running.

**Comment by TylerL95** [comment]
Hi,
i installed the version you supplied but im still not getting the recaptcha on the register page, ive added the code, enabled the plugin and the plugin to work on the registration form.
http://www.officegb.co.uk/customer/account/create/
Any idea?
Many thanks,
Tyler

**Comment by studioforty9-dev** [comment]
Did you enable it via configuration. Try turning on template/block hints to see if the block is being added.

**Comment by TylerL95** [comment]
The Block is there. It references
"frontend/base/default/template/studioforty9/recaptcha/explicit.phtml"
but it is not showing

**Comment by studioforty9-dev** [comment]
Make sure you've added the code `<?php echo $this->getChildHtml('studioforty9.recaptcha.explicit'); ?>` to the correct theme. Open up `frontend/base/default/template/studioforty9/recaptcha/explicit.phtml` and try adding 'Hello' or any text in there, see if that shows up. If it does, it's likely the isAllowed method is returning false.

**Comment by TylerL95** [comment]
Hi,
the block is not showing in the register page but it is showing up in the contact page along with the extra text that i added

**Comment by studioforty9-dev** [comment]
Can you email FTP details and access to your Magento CP to eoghan [ at ] studioforty9 [ dot ] com please?

**Comment by TylerL95** [comment]
Hello,
Ive sorted it. thank you very much for your time! :)
Tyler

**Comment by studioforty9-dev** [comment]
Great, what was the problem?

**Comment by TylerL95** [comment]
Was my fault, i was tinkering with the code before i got my reply, i reverted it and its working correctly now

**Comment by adglasser** [comment]
Still answering questions regarding this? :) I hope so!

**Comment by adglasser** [comment]
If someone is listening, out there, in the abyss. Hear my calls ...

Installed fine (R 1.5), shows up on admin, all files seem to be in place, code calling child in place inside form, but nothing happening. It's not calling the child (explicit.phtml). Added a few lines of HELLO WORLD in explicit.phtml to see if something was being seen, but nada. Hoping someone in the solar system can help.

Currently running Magento 1.6, working on new 2.2 Magento install, but need this Captcha implemented on the 1.6 for the time being (all email was shut down by host due to bot attacks on registration form).

Update: HELLO WORLD is showing up if I enter it atop the explicit.phtml. Nada on the captcha.

Update 2: Holy smokes, it works. I don't know what changed, other than me turning off and on the module about 4 times.

Update 3: It worked once, now it doesn't show up again. Bonkers.
